server/schema/posts.js

```diff
@@ -0,0 +1,42 @@
+var mongoose=require('mongoose');
+var schema=mongoose.Schema;
+ObjectId=schema.ObjectId
+var post=new schema({
+  user:{
+    type:ObjectId
+  },
+  title:{
+    type:String,
+    required:true
+  },
+  content:{
+    type:String,
+    required:true
+  },
+  image:{
+    type:String
+  },
+  Upvotes:{
+    type:Object
+  },
+  DownVotes:{
+    type:Object
+  },
+  tags:{
+    type:Array
+  },
+  Comments:{
+      type:[{user:ObjectId,
+      content:String}]
+  },
```

Comment:
https://mongoosejs.com/docs/guide.html#timestamps

Reply:
why aren't we using the default timestamp class? what will be the tradeoffs?
Lets research and change them elsewhere too.

Reply:
Okay got it

Reply:
Resolved this issue in next commit